packages/editor/src/plugins/spoiler/static.tsx

```diff
@@ -1,18 +1,40 @@
 import { StaticRenderer } from '@editor/static-renderer/static-renderer'
 import { EditorSpoilerDocument } from '@editor/types/editor-plugins'
+import { useState } from 'react'
 
 import { SpoilerRenderer } from './renderer'
+import { abSubmission } from '@/helper/ab-submission'
+import { useAB } from '@/contexts/ab'
 
 export function SpoilerStaticRenderer({
   state,
   openOverwrite,
 }: EditorSpoilerDocument & { openOverwrite?: boolean }) {
   const { title, content } = state
+  const [hasSentSpoilerTrackingEvent, setHasSentSpoilerTrackingEvent] = useState(false)
+
+  const ab = useAB()
+
+  const trackSpoilerOpened = () => {
+    if(!ab || hasSentSpoilerTrackingEvent) return
+    // send tracking event
+    abSubmission({
+      entityId: -1, // should this be a real id?
```

Comment:
This needs to be double checked.

If entity id is needed, what id is it? I see string Ids but the abSubmission expects an int